example/particle_leapfrog/particle_leapfrog.cpp

```diff
@@ -394,17 +395,7 @@ TaskStatus InitializeCommunicationMesh(const BlockList_t &blocks) {
       swarm->vbswarm->bd_var_.req_send[nb.bufid] = MPI_REQUEST_NULL;
     }
   }
-
-  for (auto &block : blocks) {
-    auto &pmb = block;
-    auto sc = pmb->swarm_data.Get();
-    auto swarm = sc->Get("my particles");
-
-    for (int n = 0; n < swarm->vbswarm->bd_var_.nbmax; n++) {
-      auto &nb = pmb->pbval->neighbor[n];
-      swarm->vbswarm->bd_var_.flag[nb.bufid] = BoundaryStatus::waiting;
-    }
-  }
```

Comment:
I assume this isn't necessary anymore?

Reply:
That the loop that is duplicated. If I didn't miss anything the exact same loop follows this one. Probably a copy/paste leftover.

Reply:
Yep just looks like a mistaken duplication